### PR TITLE
Modify email example

### PR DESCRIPTION
**Pull request by Daniel-Cretu**

### CODE REVIEW

File: app/Http/Controllers/ContactController.php [hunk 1]
```diff
@@ -2,12 +2,34 @@
...
     }
+    public function send(ContactRequest $request): RedirectResponse
+    {
+        $data = $request->validated();
+        \Log::debug('test', $data);
+        \Mail::send(
+            'molecules.emailContact',
+            [
+                'email' => $data['email'],
+                'name' => $data['name'],
```
Comment (egorbwork) [inline app/Http/Controllers/ContactController.php:22]: I requested a select value in the process.

File: resources/views/molecules/emailContact.blade.php [hunk 2]
```diff
@@ -0,0 +1,6 @@
+<!DOCTYPE html>
+<html lang="{{ str_replace('_', '-', app()->getLocale()) }}">
+<body>
+<p>Message: {{ $messageText }}</p>
+<p>From: {{ $name }}:</p>
+</body>
```
Comment (egorbwork) [inline resources/views/molecules/emailContact.blade.php]: please fix missing `</html>`

**Review by egorbwork — CHANGES_REQUESTED** [review]
Looks clean. But few adjustments are required.

File: app/Http/Controllers/ContactController.php (hunk 1 above)
Comment (Daniel-Cretu) [reply app/Http/Controllers/ContactController.php:22]: Added a select value "Subject" into contact form logic.

File: resources/views/molecules/emailContact.blade.php (hunk 2 above)
Comment (Daniel-Cretu) [reply resources/views/molecules/emailContact.blade.php]: Fixed by adding </html> closing tag.

File: resources/views/molecules/emailContact.blade.php [hunk 3]
```diff
@@ -0,0 +1,8 @@
+<!DOCTYPE html>
+<html lang="{{ str_replace('_', '-', app()->getLocale()) }}">
```
Comment (egorbwork) [inline resources/views/molecules/emailContact.blade.php:2]: Would be usefull to keep user email in this mail.

File: app/Http/Controllers/ContactController.php (hunk 1 above)
Comment (egorbwork) [reply app/Http/Controllers/ContactController.php:22]: accepted

**Review by egorbwork — APPROVED** [review]
It is acceptable but improvement could be applied.

File: resources/views/molecules/emailContact.blade.php (hunk 3 above)
Comment (Daniel-Cretu) [reply resources/views/molecules/emailContact.blade.php:2]: Modified email structure a little to fit email in it's logic
image

**Pull request merged**